ext/opentelemetry-ext-opentracing-shim/tests/test_shim.py

```diff
@@ -431,3 +443,77 @@ def test_span_on_error(self):
         self.assertEqual(
             scope.span.unwrap().events[0].attributes["error.kind"], Exception
         )
+
+    def test_inject(self):
+        """Test `inject()` method."""
+
+        otel_context = trace.SpanContext(trace_id=1220, span_id=7478)
+        context = opentracingshim.SpanContextShim(otel_context)
+
+        # Verify Format.TEXT_MAP
+        text_map = {}
+        self.shim.inject(context, opentracing.Format.TEXT_MAP, text_map)
+        self.assertEqual(text_map[MockHTTPTextFormat.TRACE_ID_KEY], str(1220))
+        self.assertEqual(text_map[MockHTTPTextFormat.SPAN_ID_KEY], str(7478))
+
+        # Verify Format.HTTP_HEADERS
+        http_headers = {}
+        self.shim.inject(
+            context, opentracing.Format.HTTP_HEADERS, http_headers
+        )
+        self.assertEqual(
+            http_headers[MockHTTPTextFormat.TRACE_ID_KEY], str(1220)
+        )
+        self.assertEqual(
+            http_headers[MockHTTPTextFormat.SPAN_ID_KEY], str(7478)
+        )
+
```

Comment:
Could we have the error as a separated test case? ;)

Reply:
I separated different formats in different test cases in https://github.com/open-telemetry/opentelemetry-python/pull/256/commits/30de89054e47c0fb2819d21c034c1a41076eb633.